### ISSUE TITLE
Zoom.us download failed

### ISSUE THREAD

**Issue by micmil** [autopkg/homebysix-recipes#337]
Since the update to 5.3 the download is failing for Zoom.us.
<img width="401" alt="Screen Shot 2020-09-22 at 11 03 35 AM" src="https://user-images.githubusercontent.com/7451137/93931966-1946c480-fce5-11ea-9ca0-2bbe7bf7ae18.png">

**Comment by jw-macworks** [comment]
Seconded. Upon investigation it seems the file path is .../AutoPkg/Cache/local.munki.Zoom/Zoom/Applications/zoom.us.app/zoom.us.app/Contents/Info.plist

The app is nested within an icon-less app package.
![Screen Shot 2020-09-22 at 1 12 28 PM](https://user-images.githubusercontent.com/57365580/93932756-85b9c780-fcd5-11ea-8292-8f854ed011ab.png)
by the way thanks for all the recipes.

**Comment by homebysix** [comment]
Is this still failing even after #334 was merged in? I believe this issue may be a duplicate of #335 and #333, which have both been resolved.

**Comment by micmil** [comment]
This appears to be resolved, thanks!
